adoc/retail_chap_install.adoc

```diff
@@ -128,59 +136,43 @@ systemctl restart salt-master
 ----
 
 .Procedure: Install the {smr} pattern on the central server
-. Step one
-. Another step
 
-.Procedure: Install and configure the build host
-. Step one
-. Another step
+. Install the {smr} pattern on the central server:
++
+----
+zypper in -t pattern pattern_name
+----
 
-.Procedure: Register the build host to {susemgr}
-. Step one
-. Another step
 
-.Procedure: Set the build host as an OS Image build host in the {susemgr} {webui}, and apply highstate
-. Standard OS Image building feature of {susemgr} is used. Images are built from customized Kiwi templates, as a starting point see SUSE provided OS Image templates.
-. Adapt these templates to contain all required software, branding, configurations. Refer to Kiwi Image System documentation for details how to work with Kiwi templates.
-. If you are creating Kiwi template from scratch, please make sure image/preferences/type elements attribute boot is set to saltboot/suse-SLES12
-. Once the image is build, it will be automatically transferred to {susemgr} server
+.Procedure: Install the build host and register it to {susemgr}
 
-.Procedure: Install and configure the branch server
-TODO: https://github.com/SUSE/spacewalk/issues/5611
-. Step one
-. Another step
+Your build host should be a Salt minion, running {sls}{nbsp}12, SP{nbsp}3.
```

Comment:
we write `{sls}{nbsp}12 SP3`, that no comma and no break-space between SP and the version number. I do not know why, but we do this this quite some time and ist looks fine, IMO.